docs/source/notebooks/gridded_data_subset.ipynb

```diff
@@ -205,81 +202,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
```

Comment:
notebook halted execution in the middle?

Reply:
A failure stopped the execution.

Reply:
Hopefully the new timeout value fixes this.